stdlib/public/core/StringBridge.swift

```diff
@@ -250,11 +250,6 @@ public final class _NSContiguousString : _SwiftNativeNSString {
     }
   }
 
-  @objc
-  func _fastCharacterContents() -> UnsafeMutablePointer<UInt16>? {
-    return _core.elementWidth == 2 ? _core.startUTF16 : nil
-  }
-
   //
```

Comment:
my thought was that things that have to do with Foundation should belong in the Foundation overlay, but I didn't want to move the entire class since that would be a much larger change